src/pages/country.js

```diff
@@ -0,0 +1,54 @@
+import { useQuery, gql } from "@apollo/client";
+
+export const COUNTRY_QUERY = gql`
+  query getCountry($Id: ID!) {
+    countries(where: { id: { eq: $Id } }) {
+      name
+      vatRate
+      id
+      capital {
+        name
+      }
+      cities {
+        name
+      }
+    }
+  }
+`;
+
+const Country = (props) => {
+
+  const { loading, error, data } = useQuery(COUNTRY_QUERY, {
+    variables: { Id: props.match.params.countryId },
+  });
+
+  if (loading) return <div>Loading...</div>
+  if (error) return <div>something is wrong</div>
+  console.log('dupa') // ;-)
+  
+  return (
+    <div>
+      {data ?
+        (data.countries.map((country, index) => (
```

Comment:
We don't need index

Reply:
I am wondering if in the case that we are sure that this is a  `one` country cause we are looking it through `id` maybe a better option would be to do `countries[0]` instead of mapping which suggests the collection? It can be misleading during reading of the code

Reply:
Yeah, it's much better. I swapped mapping through array with one object, for having data directly from `countries[0]`